service/src/main/java/uk/gov/hmcts/reform/fpl/model/Other.java

```diff
@@ -158,4 +158,19 @@ public boolean isDeceasedOrNFA() {
         return AddressNotKnowReason.DECEASED.getType().equals(addressNotKnowReason)
             || AddressNotKnowReason.NO_FIXED_ABODE.getType().equals(addressNotKnowReason);
     }
+
+    public Other removeAddress() {
+        Other other =  this.toBuilder()
```

Comment:
You can just return the builder
`return this.toBuilder()
.address(null)
.build();
`

Reply:
changed